src/code/dialogs/opsDialog.js

```diff
@@ -194,20 +214,22 @@ const OpsDialog = WDialog.extend({
                 alert(e);
               }
             }
-            const ol = opsList();
-            let newopID = ol[0];
-            if (newopID == null || newopID == selectedOp.ID) {
-              console.log(
-                "removing first op in list? I was going to use that...."
-              );
-              newopID = ol[1];
-              if (newopID == null) {
-                console.log("not removing last op... fix this");
-                // create a new default op and use that -- just call the init/reset functions?
-              }
-            }
-            const removeid = selectedOp.ID;
-            const newop = makeSelectedOperation(newopID);
+            removeOperation(selectedOp.ID);
+            const newop = changeOpIfNeeded();
+            // const ol = opsList();
+            // let newopID = ol[0];
+            // if (newopID == null || newopID == selectedOp.ID) {
+            //   console.log(
+            //     "removing first op in list? I was going to use that...."
+            //   );
+            //   newopID = ol[1];
+            //   if (newopID == null) {
+            //     console.log("not removing last op... fix this");
+            //     // create a new default op and use that -- just call the init/reset functions?
+            //   }
+            // }
+            // const removeid = selectedOp.ID;
+            // const newop = makeSelectedOperation(newopID);
```

Comment:
you can go ahead and remove the commented out stuff.

Reply:
done